### PR TITLE
Updated /src/.env-example

### PR DESCRIPTION
**Pull request by nirvanadev**
Thanks for sharing such an awesome setup

### CODE REVIEW

**Comment by urre** [comment]
@nirvanadev thanks. But the `./src/enc` don't need these variables `APP_NAME, DOMAIN
DB_ROOT_PASSWORD`, they are only used for docker-compose, nginx and phpmyadmin services. So feels redundant to include them in that env file also

**Comment by nirvanadev** [comment]
Okay, no worries. They were "Warnings" and not errors I received when running the initial `docker-compose run composer install`

**Comment by nirvanadev** [comment]
I did need to remove the `:3306` part from mysql in the one commit in order for this to work. Not sure why. I'm on OSX Catalina.

**Comment by urre** [comment]
@nirvanadev Ok, thanks. Yes, you're right about the port number. That looks correct, I'm on Mojaka still so the current config works for me. I'll test it a bit tomorrow.

File: Dockerfile [hunk 1]
```diff
@@ -5,5 +5,5 @@ RUN curl -o /bin/wp-cli.phar https://raw.githubusercontent.com/wp-cli/builds/gh-
 RUN chmod +x /bin/wp-cli.phar
 RUN cd /bin && mv wp-cli.phar wp
 RUN mkdir -p /var/www/.wp-cli/cache && chown www-data:www-data /var/www/.wp-cli/cache
-
+RUN wp core install --url=http://compulsev2.local --title=radcli --admin_user=compulse_support --admin_password=2cSbmQWQOyB6QS1Sfh --admin_email=krwilliams@sbgtv.com --allow-root
```
Comment (urre) [inline Dockerfile]: @nirvanadev ?

**Comment by urre** [comment]
@nirvanadev thanks for you contribution!

**Pull request merged**